internal/credentials/credentials.go

```diff
@@ -71,9 +71,9 @@ type Database interface {
 // utilizes a Kubernetes controller runtime client to retrieve credentials
 // stored in Kubernetes Secrets.
 type kubernetesDatabase struct {
-	kargoClient client.Client
-	argoClient  client.Client // nil if credential borrowing is not enabled
-	cfg         KubernetesDatabaseConfig
+	kargoClient  client.Client
+	argocdClient client.Client // nil if credential borrowing is not enabled
```

Comment:
You see changes like this throughout. "argoClient" was too ambiguous, because Rollouts is also and Argo project. `argocdClient` is more specific.